src/app/Page.js

```diff
@@ -11,15 +13,75 @@ export default class Page extends React.Component<{|+children: Node|}> {
     return (
       <div>
         <header>
-          <nav>
-            <ul style={{listStyle: "none", paddingLeft: 0, margin: 0}}>
+          <nav className={css(style.nav)}>
+            <ul className={css(style.navList)}>
+              <li className={css([style.navItem, style.navItemLeft])}>
```

Comment:
nit: I believe that this is equivalent to

```js
css(style.navItem, style.navItemLeft)
```

(i.e., without the array). If they’re equivalent, I have a weak
preference toward removing the array, because it makes the single-style
case more consistent with the multi-style case, which in turn minimizes
churn.